nonebot/command/__init__.py

```diff
@@ -203,25 +208,41 @@ def __str__(self):
 
 
 class CommandManager:
-    """Global Command Manager"""
+    """全局命令管理器。
+
+    版本: 1.6.0+
+    """
     _commands = {}  # type: Dict[CommandName_T, Command]
     _aliases = {}  # type: Dict[str, Command]
     _switches = {}  # type: Dict[Command, bool]
     _patterns = {}  # type: Dict[Pattern[str], Command]
 
     def __init__(self):
         self.commands = CommandManager._commands.copy()
+        """命令字典。"""
```

Comment:
mark

Reply:
> mark. I will fix

Pull the newest nb-autodoc to generate API doc.

Detail doc change from nb-autodoc: https://github.com/iyume/nonebot/commit/449aec16df8068f6045407c4d3615c5622fcd2ed